packages/atomic-angular/projects/atomic-angular/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Revert those dependency updates; 3.1.6 is the latest atomic-angular version (3.1.7 is a faulty v2 release).